packages/graph/hash_graph/lib/graph/src/types/schema/entity_type.rs

```diff
@@ -155,6 +158,8 @@ impl EntityType {
 
 #[cfg(test)]
 mod tests {
```

Comment:
I don't think I'm seeing tests to check we can _update_ types with a new version? (As well as checking it fails as expected if you try and publish a different type with the same URI)

Reply:
Tests for updating does not happen in `EntityType` but in the database in `datastore::postgres::tests`.

Reply:
Sorry yes that was silly of me, the comment was in the incorrect place. I still don't think our test-cases are checking enough of the space, I don't immediately see a test for this:
> As well as checking it fails as expected if you try and publish a different type with the same URI

Reply:
Alright, I'm okay with us deferring but generally it'd be good to write tests as we go. We're moving fast but we should be careful to not accrue too much tech debt